homeassistant/components/voip/config_flow.py

```diff
@@ -22,7 +35,10 @@ async def async_step_user(
             return self.async_abort(reason="single_instance_allowed")
 
         if user_input is None:
-            return self.async_show_form(step_id="user")
+            return self.async_show_form(
+                step_id="user",
+                data_schema=STEP_USER_DATA_SCHEMA,
```

Comment:
We should only show this if a user is running in advanced mode. Most users shouldn’t care. 

The other reason I am not a fan of this option is because it makes the tutorial harder. What is driving this feature?

Reply:
Port collisions

Reply:
How about moving this into an options flow instead? We can keep the initial setup clean of it that way.